docs/version.asciidoc

```diff
@@ -1,33 +1,11 @@
-// stack-version can be:    7.0.1, 8.0.0-alpha1, 8.1.0, etc.
-// major-version can be:    8.x, 7.x, 6.x, etc.
-// doc-branch can be:       master, 8.0, 8.1, etc.
-// release-state is only:   released | prerelease | unreleased
+// Use shared version file
+include::{asciidoc-dir}/../../shared/versions/stack/{source_branch}.asciidoc[]
+
+// Docs copied from Beats use stack-version and doc-branch instead of version
+:stack-version: {version}
```

Comment:
These two lines need to be updated to `{apm_server_version}` before this is merged. I'm using `{version}` for now to make the docs ci point out broken links.

Reply:
After a bit more digging. This needs to stay as-is right now. I can only update to `{apm_server_version}` after https://github.com/elastic/docs/pull/1187 is merged.